src/companies/views/RegisterCompany.vue

```diff
@@ -0,0 +1,139 @@
+                class="mb-12"
+                height="400px"
+                width="800px"
+                color="#f8f9fa"
+              >
+                <v-card-title class="fs35 colorOrange mb-6">
+                  Test Company
+                </v-card-title>
+                <v-divider></v-divider>
+                <v-divider></v-divider>
+                <v-divider></v-divider>
+
+                <v-card-text class="fs20 mt-3 mb-15">
+                  Do you want to test our services using a demo company? Use
+                  this option do that. We will create a demo company for you
+                  where you may try out our services without any effect on your
+                  real company.
+                </v-card-text>
+
+                <v-card-actions>
+                  <v-btn
+                    x-large
+                    @click="e1 = 2"
+                    class="ma-2"
+                    color="#f2b334"
+                    outlined
+                  >
+                    REGISTER A TEST COMPANY
+                  </v-btn>
+                </v-card-actions>
+              </v-card>
+              <v-divider></v-divider>
+            </v-stepper-content>
+
+            <v-stepper-content step="2">
+              <p class="fs40 pdbt-40">Register a test company</p>
+              <p class="fs20">
+                Please fill out the below to set up your company profile.<br />
+                The name and organization number will be auto-generated.
+              </p>
+              <RegisterForm @register-company="register" />
+            </v-stepper-content>
+
+            <v-stepper-content step="3">
+              <p class="fs40 pdbt-40">Company created successfully</p>
+              <p class="mb-12 fs20">
+                discreet desk is registered successfully and ready to be used.
+                Click here if you want to go to
+                <a href="" style="text-decoration: none">discreet desk</a>
+              </p>
+              <v-btn
+                color="#72418b"
+                dark
+                class="mr-5"
+                x-large
+                href="/companies"
+              >
+                COMPANY LIST
+              </v-btn>
+              <v-btn color="#f2b334" dark x-large @click="e1 = 1">
+                CREATE NEW COMPANY
+              </v-btn>
+            </v-stepper-content>
+          </v-stepper-items>
+        </v-stepper>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import RegisterForm from "../components/RegisterCompanyForm.vue";
+export default {
+  components: {
+    RegisterForm,
+  },
+  data() {
+    return {
+      e1: 1,
+    };
+  },
+  methods: {
+    register(value) {
+      this.e1 = 3;
+      this.$store.dispatch("companies/register", {
+        name: value.name,
+        phoneNumber: value.phoneNumber,
+        address: value.address,
+        website: value.website,
+        currency: value.currency,
+      });
```

Comment:
```suggestion
    register(value) {
      this.e1 = 3;
      this.$store.dispatch("companies/register", value);
```

We don't need to create a new object again